website/content/docs/upgrading/version-guides/0.9.0.mdx

```diff
@@ -0,0 +1,169 @@
+---
+layout: docs
+page_title: Version 0.9.0 - Upgrade Guide
+description: |-
+  This upgrade guide describes any considerations for upgrading to Waypoint 0.9.0.
+---
+
+# Upgrading to Waypoint 0.9.0 from 0.8.x
+
+Waypoint 0.8.x can be upgraded to Waypoint 0.9.0 using a [standard upgrade](/docs/upgrading#standard-upgrade).
+There are some considerations depending on what platform your server is installed on.
+There are no breaking changes in the protocol or `waypoint.hcl` configuration file.
+Due to the changes in `runner` management, we cannot guarantee that all Waypoint
+components (CLI, entrypoint, and server) are forwards and backwards
+compatible. We encourge users to have all components on the same version.
+
+## ECS Upgrades
+
+During prerelease testing, we discovered an issue that has exited since the 0.8.1
+release, and possibly earlier, that means `waypoint upgrade` does not work on
+servers installed on ECS. We found this too late to fix it for the 0.9.0 release,
+but plan to fix it for a future release. You can track [the issue here](https://github.com/hashicorp/waypoint/issues/3527).
+
+If you want to upgrade your Waypoint server installed on ECS, you will need to
+snapshot your sever, do a Waypoint server uninstall, complete a new express
+install with v0.9.0, and restore your server snapshot. Note that the
+`waypoint uninstall` command creates a snapshot by default.
+An example is in the below [Kubernetes Upgrades](#kubernetes-upgrades) section.
+
+### Change to Region Flag
+
+We removed the default value for the flag `-ecs-region`. This was previously set
+to `us-west-2`. If you used the default on the install of Waypoint to ECS, you will
+need to specify `us-west-2` moving forward.
+
+## Kubernetes Upgrades
+
+With 0.9 comes a significant change in how Waypoint manages express server installs.
+Waypoint now uses Helm under the hood to all Waypoint components. If you previously
+installed with Helm, you can run a standard Waypoint upgrade.
+
+If you previously installed with the Waypoint express install for Kubernetes,
+you will need to snapshot your sever, do a Waypoint server uninstall, complete a
+new express install with v0.9.0, and restore your server snapshot. Note that the
+`waypoint uninstall` command does creates a snapshot by default. It is also vitally
+important that you set `-delete-context=false` on the uninstall.
```

Comment:
```suggestion
`waypoint uninstall` command does create a snapshot by default. **It is also vitally
important that you set `-delete-context=false` on the uninstall.**
```

Reply:
oh I did a sneaky thing where I deleted the first review and tried to combine with another lol. We could bold the vitally important bit for emphasis